lambdas/enums/repository_role.py

```diff
@@ -9,4 +9,4 @@ class RepositoryRole(Enum):
 
     @staticmethod
     def list():
-        return [org.value for org in list(RepositoryRole)]
+        return tuple(item.value for item in RepositoryRole)
```

Comment:
If you're returning a tuple instead of a list better to change the fun name from 'list'

Reply:
Done